.github/workflows/sycl-rel-nightly.yml

```diff
@@ -0,0 +1,272 @@
+      target_devices: ${{ matrix.target_devices }}
+      tests_selector: ${{ matrix.tests_selector }}
+      extra_lit_opts: ${{ matrix.extra_lit_opts }}
+      reset_intel_gpu: ${{ matrix.reset_intel_gpu }}
+      ref: ${{ env.SYCL_REL_BRANCH }}
+      merge_ref: ''
+      sycl_toolchain_artifact: sycl_linux_default
+      sycl_toolchain_archive: ${{ needs.ubuntu2204_build.outputs.artifact_archive_name }}
+      sycl_toolchain_decompress_command: ${{ needs.ubuntu2204_build.outputs.artifact_decompress_command }}
+
+  build-win:
+    needs: [check_for_new_commits]
+    if: ${{ github.repository == 'intel/llvm' && needs.check_for_new_commits.outputs.is_new_commit != 'false' }}
+    uses: ./.github/workflows/sycl-windows-build.yml
+    with:
+      retention-days: 90
+      # We upload both Linux/Windows build via Github's "Releases"
+      # functionality, make sure Linux/Windows names follow the same pattern.
+      artifact_archive_name: sycl_windows.tar.gz
+      build_ref: ${{ env.SYCL_REL_BRANCH }}
+
+  e2e-win:
+    needs: build-win
+    # Continue if build was successful.
+    if: |
+      always()
+      && !cancelled()
+      && needs.build-win.outputs.build_conclusion == 'success'
+    uses: ./.github/workflows/sycl-windows-run-tests.yml
+    with:
+      name: Intel GEN12 Graphics with Level Zero
+      runner: '["Windows","gen12"]'
+      sycl_toolchain_archive: ${{ needs.build-win.outputs.artifact_archive_name }}
+      extra_lit_opts: --param gpu-intel-gen12=True
+      ref: ${{ env.SYCL_REL_BRANCH }}
+
+  cuda-aws-start:
+    needs: [ubuntu2204_build]
+    if: ${{ always() && !cancelled() && needs.ubuntu2204_build.outputs.build_conclusion == 'success' }}
+    uses: ./.github/workflows/sycl-aws.yml
+    secrets: inherit
+    with:
+      mode: start
+      ref: ${{ env.SYCL_REL_BRANCH }}
+
+  cuda-run-tests:
+    needs: [ubuntu2204_build, cuda-aws-start]
+    if: ${{ always() && !cancelled() && needs.ubuntu2204_build.outputs.build_conclusion == 'success' }}
+    uses: ./.github/workflows/sycl-linux-run-tests.yml
+    with:
+      name: CUDA E2E
+      runner: '["aws_cuda-${{ github.run_id }}-${{ github.run_attempt }}"]'
+      image: ghcr.io/intel/llvm/ubuntu2204_build:latest-0300ac924620a51f76c4929794637b82790f12ab
+      image_options: -u 1001 --gpus all --cap-add SYS_ADMIN --env NVIDIA_DISABLE_REQUIRE=1
+      target_devices: ext_oneapi_cuda:gpu
+      ref: ${{ env.SYCL_REL_BRANCH }}
+      merge_ref: ''
+
+      sycl_toolchain_artifact: sycl_linux_default
+      sycl_toolchain_archive: ${{ needs.ubuntu2204_build.outputs.artifact_archive_name }}
+      sycl_toolchain_decompress_command: ${{ needs.ubuntu2204_build.outputs.artifact_decompress_command }}
+
+  cuda-aws-stop:
+    needs: [cuda-aws-start, cuda-run-tests]
+    if: always()
+    uses: ./.github/workflows/sycl-aws.yml
+    secrets: inherit
+    with:
+      mode: stop
+      ref: ${{ env.SYCL_REL_BRANCH }}
+
+  # No binaries upload for now
+  # nightly_build_upload:
```

Comment:
can we remove commented code?

Reply:
@AlexeySachkov will we even need this in the future? I'm not sure so may be it can be actually dropped.

Reply:
Removed this part - [e39dae8](https://github.com/intel/llvm/pull/16297/commits/e39dae8465d8c7c9aee7b96e49c05f773d7ad9a7)